examples/istio-with-separate-cert/README.md

```diff
@@ -0,0 +1,105 @@
+# Example: Big Bang's Istio with a separately loaded cert
+
+This example deploys Big Bang's Istio, but without an ingress cert. It is applicable in use cases where you want to have a freely distributable zarf package, but your ingress cert is private and can't be distributed in the same way that you want the Zarf package to be.
+
+## Known Issues
+
+The same known issues that are documented in [the Big Bang example](../big-bang/README.md#known-issues) apply here as well, except for the Elasticsearch stuff since we have EFK turned off.
+
+## Prerequisites
+
+1. Install [Vagrant](https://www.vagrantup.com/)
+1. Install `make`
+1. Install `sha256sum` (on Mac it's `brew install coreutils`)
+1. [Logged into registry1.dso.mil](https://github.com/defenseunicorns/zarf/blob/master/docs/ironbank.md#2-configure-zarf-the-use-em)
+
+## Instructions
+
+### Pull down the code and binaries
+
+```shell
+# clone the binaries
+git clone https://github.com/defenseunicorns/zarf.git
+
+# change to the examples folder
+cd zarf/examples
+
+# Download the latest release of Zarf and the Init Package to the 'examples/sync' folder
+make fetch-release
+```
+
+> NOTE:
+>
+> If you have any issues with `make fetch-release` you can try `make build-release` instead. It will build the files instead of downloading them. You'll need Golang installed.
+
+### Build the deploy package
+
+```shell
+# Create the deploy package and move it to the 'examples/sync' folder. It will also create a kubernets manifest with the '*.bigbang.dev' cert that you can use later in the example.
+make package-example-istio-with-separate-cert
+```
+
+### Start the Vagrant VM
+
+```shell
+# Start the VM. You'll be dropped into a shell in the VM as the Root user
+make vm-init
+```
+
+> NOTE:
+>
+> All subsequent commands should be happening INSIDE the Vagrant VM
+
+### Initialize Zarf
+
+```shell
+# Initialize Zarf
+./zarf init --confirm --components k3s,gitops-service
```

Comment:
This assumes appliance mode, is that the only support mode for this example?  I'd much rather not lean on makefiles, vagrant, appliance mode when doing any of the examples as I think it mixes concerns.

Reply:
It's possible that this example would work in KinD, but most Big Bang examples do not due to issues with Elasticsearch. I would rather leave this using the same pattern for now rather than trying to do something different.

Side-note: What is your definition of Appliance Mode? I thought it was "appliance mode" when you don't use the gitops-service.

Reply:
okay thanks and yes we need to sort out #174 pretty soon too I think.  I've used "appliance mode" in multiple ways and generally it was something without the gitops service, but have recently been using it for the version of zarf that also deploys k3s since we control all of it--but that's probably not totally the right term.